.github/workflows/build_sample.yml

```diff
@@ -30,12 +30,16 @@ jobs:
     - name: Validate Gradle wrapper
       uses: gradle/wrapper-validation-action@v1.0.5
 
-
     - name: Decrypt the keystore for signing
       run: |
         echo "${{ secrets.KEYSTORE_ENCRYPTED }}" > keystore.asc
         gpg -d --passphrase "${{ secrets.KEYSTORE_PASSWORD }}" --batch keystore.asc > keystore.jks
 
+    - name: Download and extract `ark-core` JNI libs
+      run: |
+        wget https://github.com/ARK-Builders/ark-core/releases/download/v1.0.0/jniLibs.zip
+        unzip -d sample/src/main jniLibs.zip
+
```

Comment:
Sorry for suddenly jumping into this, but I think `implementation(libraries.core)` is enough. In theory we don't have to manually download the JNI libs like this because it's already included in the synced dependency of `arkcore`.

Reply:
Just repeating what we later discussed: `implementation(libraries.core)` is not currently enough, since `ark-core/java` is a JAR, that means JNI libs are not available for use from it